qa/wildfly/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -3,5 +3,8 @@
         <exclusions>
             <module name="org.jboss.resteasy.resteasy-jackson2-provider" />
         </exclusions>
+        <dependencies>
```

Comment:
Not really related to this PR, but do you think this `wildfly` subproject is adding value? According to the developer guide, this exists to test clients in "interesting" places so I suppose it is somewhat relevant here. Would you consider adding HTTP/2 or streaming specific tests to this package? If not, I'm going to question its value :)

The primary overhead of this subproject is that it brings in dependencies like resteasy that otherwise wouldn't be here.

Reply:
> Would you consider adding HTTP/2 or streaming specific tests to this package? If not, I'm going to question its value :)

I am working on test cases as we speak :) I think we could definitely add streaming tests there, thanks a lot for the idea @andrross !

Reply:
Covered by `qa/smoke-test-http` now